pyirf/interpolation/utils.py

```diff
@@ -0,0 +1,164 @@
+import numpy as np
+
+
+def plumb_point_dist(line, target):
```

Comment:
any references for the formulas here?

Reply:
Sure, I've found a wiki entry, hope that suffices. You can find them below, links were also added as comments to the code 
https://en.wikipedia.org/wiki/Distance_from_a_point_to_a_line#Another_formula

Reply:
> This derivation also requires that the line be not vertical or horizontal. 

Do we handle that case?

Reply:
This caveat is, as I see it, only for the analytical deviation, not for the vector ones I used. I messed up the link in the comment above (sorry for that :sweat_smile:), it should redirect to the section after the one it does link to (https://en.wikipedia.org/wiki/Distance_from_a_point_to_a_line#Vector_formulation). The links in the code-comments however are correct. To check things, I've added additional test cases and these cases are as far as the tests go handled correctly.